password_security/controllers/main.py

```diff
@@ -44,6 +45,7 @@ def web_login(self, *args, **kw):
             request.params['password']
         )
         if not uid:
+            request.uid = old_uid
```

Comment:
How can we do like that? When we reference to `base.public_user`, it means we access to `ir.model.data`, and sql query will be  executed within access right checking. At that time, still no uid so it wil cause error